### ISSUE TITLE
Wizard FatalError

### ISSUE THREAD

**Issue by Jduret** [phpro/soap-client#347]
### Bug Report

|    Q        |   A
|------------ | ------
| BC Break    | don't know
| Version     | 1.3.2

#### Summary

I'm on Windows10 with PHP7.2.9.
Using the Wizard, a Fatal error occured during generation process.
So I'm unable to use the Wizard.

#### Current behavior

The error is :
```
PHP Fatal error:  Uncaught Error: Call to a member function setNamespaceName() on boolean in C:\projects\my-project\vendor\phpro\soap-client\src\Phpro\SoapClient\CodeGenerator\TypeGenerator.php:49
Stack trace:
#0 C:\projects\my-project\vendor\phpro\soap-client\src\Phpro\SoapClient\Console\Command\GenerateTypesCommand.php(201): Phpro\SoapClient\CodeGenerator\TypeGenerator->generate(Object(Laminas\Code\Generator\FileGenerator), Object(Phpro\SoapClient\CodeGenerator\Model\Type))
#1 C:\projects\my-project\vendor\phpro\soap-client\src\Phpro\SoapClient\Console\Command\GenerateTypesCommand.php(132): Phpro\SoapClient\Console\Command\GenerateTypesCommand->generateType(Object(Laminas\Code\Generator\FileGenerator), Object(Phpro\SoapClient\CodeGenerator\TypeGenerator), Object(Phpro\SoapClient\CodeGenerator\Model\Type), Object(SplFileInfo))
#2 C:\projects\my-project\vendor\phpro\soap-client\src\Phpro\SoapClient\Console\Command\Ge in C:\projects\my-project\vendor\phpro\soap-client\src\Phpro\SoapClient\CodeGenerator\TypeGenerator.php on line 49
```
After some research, I found that `FileGenerator::getClass` return a boolean in my case but I don't search enough to find why.
To make the wizard running, I've just change a piece of code in `TypeGenerator::generate` and `ClientGenerator::generate` methods.
By adding these ternary in the try catch block, the problem is probably not fixed but patched for my case.

```php
$class = $file->getClass() ?: new ClassGenerator();
 ```
With this, the Wizard run normaly and my classes are well created.

#### How to reproduce

PHP7.2.9
Windows 10
Standard WSDL (can't paste it sorry)
Just execute the wizard.

#### Expected behavior

Wizard would create all Types and Client classes.

**Comment by janvernieuwe** [comment]
Hi @Jduret ,

This was fixed in version `1.4.1` can you try that version (or later) and see if that solves your issue?

**Comment by Jduret** [comment]
> Hi @Jduret ,
> 
> This was fixed in version `1.4.1` can you try that version (or later) and see if that solves your issue?

Yes I will try it during the day.

**Comment by Jduret** [comment]
Ok, with the version 1.4.1, the wizard work properly.
